static/gsApp/components/ai/AiSetupDataConsent.tsx

```diff
@@ -1,40 +1,36 @@
 import styled from '@emotion/styled';
 
-import {Button, LinkButton} from 'sentry/components/core/button';
+import autofixSetupImg from 'sentry-images/features/autofix-setup.svg';
+
+import {promptsUpdate} from 'sentry/actionCreators/prompts';
+import {SeerIcon} from 'sentry/components/ai/SeerIcon';
+import {Flex} from 'sentry/components/container/flex';
+import {Button} from 'sentry/components/core/button';
+import {useAutofixSetup} from 'sentry/components/events/autofix/useAutofixSetup';
+import ExternalLink from 'sentry/components/links/externalLink';
 import LoadingIndicator from 'sentry/components/loadingIndicator';
-import {t} from 'sentry/locale';
+import {t, tct} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 import {useMutation, useQueryClient} from 'sentry/utils/queryClient';
 import useApi from 'sentry/utils/useApi';
 import useOrganization from 'sentry/utils/useOrganization';
 
-import withSubscription from 'getsentry/components/withSubscription';
-import {useGenAiConsentButtonAccess} from 'getsentry/hooks/genAiAccess';
-import type {Subscription} from 'getsentry/types';
-
 type AiSetupDataConsentProps = {
   groupId: string;
 };
 
-function AiSetupDataConsent({
-  groupId,
-  subscription,
-}: AiSetupDataConsentProps & {
-  subscription: Subscription;
-}) {
-  const api = useApi();
+function AiSetupDataConsent({groupId}: AiSetupDataConsentProps) {
+  const api = useApi({persistInFlight: true});
   const organization = useOrganization();
   const queryClient = useQueryClient();
-
-  const endpoint = `/organizations/${organization.slug}/data-consent/`;
+  const {data: autofixSetupData} = useAutofixSetup({groupId});
 
   const genAiConsentMutation = useMutation({
     mutationFn: () => {
-      return api.requestPromise(endpoint, {
-        method: 'PUT',
-        data: {
-          genAIConsent: true,
-        },
+      return promptsUpdate(api, {
+        organization,
+        feature: 'seer_autofix_setup_acknowledged',
+        status: 'dismissed',
```

Comment:
idk if you still want to call all of this stuff "consent"

Reply:
i think I was avoiding calling it consent since we'll remove the other consent

Reply:
right, I meant that since we're removing consent, should stuff like `genAiConsentMutation` be renamed?

Reply:
yes!